.vsts.pipelines/jobs/ci-linux.yml

```diff
@@ -93,6 +94,16 @@ jobs:
         /p:ProdConBlobFeedUrlPrefix=$(prodConBlobFeedUrlPrefix)
     displayName: Run smoke-test
 
+  # Run unit tests that we support.
```

Comment:
Where do we see the result of these unit tests?

Reply:
The results for each respective project ends up being in corefx artifacts/bin under source-build. They generate a testresults.xml file.

Reply:
It'd be nice if we could upload these as artifacts in the build, don't block on it though.

Reply:
That makes sense. Would you mean copying all the testResults.xml files from coreFX artifacts to source-build root bin directory? Perhaps create some directory in source-build/bin and copy all of them over there?

Reply:
Yeah, I don't think the build generally has access to artifacts in the docker container so we might have to copy them out to bin/logs or something, but I'm fine with adding this later and just getting this in for now so we at least get an overall pass/fail.

Reply:
Sure! I can start an issue to track this as this PR gets merged.